### DISCUSSION TITLE
Pass feature flags to the underlying running rust code

### DISCUSSION THREAD

**Discussion by Acentelles** [rusterlium/rustler#664] [Q&A] [answered]
I'm running `mix test` to test some rust code with rustler.

For this rust code to run, I need to pass the `prove` feature flag like this: `cargo run -- --prove`. 

Is there a way to pass this feature flag with rustler?

**Comment by filmor** [comment]
`mix test` on a Rustler project will only ever run `cargo build`. If you mean `cargo build --feature=prove`, that is supported as a parameter on `use Rustler`.

What are you trying to do?

**Comment by Acentelles** [comment]
Hey! Thanks for your answer!

> What are you trying to do?

I'm trying to _run_ the rust code using the command flag `prove` (which can only be passed on _run_ but not on _build_)

I tried something like this:
`use Rustler, otp_app: :risc0, crate: :risc0_prover, features: :prove` but it doesn't work.

**Reply by filmor** [reply, marked as answer]
Rustler doesn't run executables. It builds a library that is then loaded by the BEAM. There are no command-line flags.
